pysumma/pysumma/Plotting.py

```diff
@@ -0,0 +1,111 @@
+import seaborn as sns
+import xarray as xr
+import numpy as np
+import matplotlib as mpl
+import matplotlib.pyplot as plt
+import pandas as pd
+
+class Plotting:
+
+	colors = ["windows blue", "amber", "greyish", "faded green", "dusty purple"]
+	sns.set_palette(sns.xkcd_palette(colors))
+
+	def __init__(self, filepath):
+		self.filepath = filepath
+
+	def open_netcdf(self):
+		filepath = self.filepath
+		ds = xr.open_dataset(filepath)
+		print(ds)
+		return ds
+
+	def plot_1d(self, ds, varibale_num):
+		var_dict= dict(ds.data_vars)
+		var_list_key = list(var_dict.keys())[varibale_num]
+		plot = ds[var_list_key].plot()
+		plt.ylabel('{} ({})'.format(ds[var_list_key].long_name, ds[var_list_key].units))
+		plt.show()
+		return plt.show()
+
+	def plot_1d_hru(self, ds, hru_num, varibale_num):
+		var_dict= dict(ds.data_vars)
+		var_list_key = list(var_dict.keys())[varibale_num]
+		plot = ds[var_list_key][:,hru_num].plot()
+		plt.ylabel('{} ({})'.format(ds[var_list_key].long_name, ds[var_list_key].units))
+		plt.show()
+		return plt.show()
+
+	def plot_1d_layer(self, ds, hru_num, varibale_num_Y, start_index_X): #='mLayerVolFracWat', start_index='midTotoStartIndex'
+		var_dict= dict(ds.data_vars)
+		var_list_key_Y = list(var_dict.keys())[varibale_num_Y]
+		plot = ds[var_list_key_Y][:,hru_num].plot()
+		timestep = 2207
+		layers = ds.nLayers[timestep].values[0]  # extract the number of layers associated with the first timestep
+		var_list_key_X = list(var_dict.keys())[start_index_X]
+		startIndex = ds[var_list_key_X][timestep].values[0] - 1  # - 1 since the SUMMA indices are 1-based and python indices are 0-based
```

Comment:
Try to avoid mixing `snake_case` and `camelCase` within a codebase.  Should be `start_index`